src/extras.jl

```diff
@@ -69,8 +71,8 @@ also accept them.
   the intervals; or a function `f(from, to, i; leftclosed, rightclosed)` that generates
   the labels from the left and right interval boundaries and the group index. Defaults to
   `"[from, to)"` (or `"[from, to]"` for the rightmost interval if `extend == true`).
-* `allowmissing::Bool=true`: when `true`, values outside of breaks result in missing values.
-  only supported when `x` accepts missing values.
+* `allowoutside::Bool=false`: when `true`, values outside of breaks result in missing values.
```

Comment:
I'm not sure what would be the best name for this argument. Note that with R's `cut` the default is `true`, which I find dangerous. And indeed `Hmisc::cut2` automatically extends provided breaks to avoid that (equivalent to our `extend=true`).

Reply:
I think it is OK, but maybe also add what is the consequence of `false` value?